### PR TITLE
Added DtoModels

### PR DESCRIPTION
**Pull request by isfandiyoro5**

### CODE REVIEW

File: TestHotel/TestHotel.DataAccess/DTO/RequestDto/EmployeeRequestDto.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        public Gender Gender { get; set; }
+
+        public string PhoneNumber { get; set; }
+
+        public string Email { get; set; }
+
+        public string Password { get; set; }
+
+        public int Salary { get; set; }
```
Comment (SheraliErkayev) [inline TestHotel/TestHotel.DataAccess/DTO/RequestDto/EmployeeRequestDto.cs]: Salary double bolishi kerak. Modelni ham o'zgartitirsh kerak shuni doublega

File: TestHotel/TestHotel.DataAccess/DTO/ResponseDto/EmployeeResponseDto.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        public int RoleID { get; set; }
+
+        public string FullName { get; set; }
+
+        public string PhoneNumber { get; set; }
+
+        public string Email { get; set; }
+
+        public int Salary { get; set; }
```
Comment (SheraliErkayev) [inline TestHotel/TestHotel.DataAccess/DTO/ResponseDto/EmployeeResponseDto.cs]: int Salary => double Salary

File: TestHotel/TestHotel.DataAccess/DTO/ResponseDto/GuestResponseDto.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestHotel.DataAccess.Model;
+
+namespace TestHotel.DataAccess.DTO.ResponseDto
+{
+    internal class GuestResponseDto
```
Comment (SheraliErkayev) [inline TestHotel/TestHotel.DataAccess/DTO/ResponseDto/GuestResponseDto.cs]: public qilish kerak hamma dtolarni

File: TestHotel/TestHotel.DataAccess/DTO/ResponseDto/RoomTypeResponseDto.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.DTO.ResponseDto
+{
+    internal class RoomTypeResponseDto
```
Comment (SheraliErkayev) [inline TestHotel/TestHotel.DataAccess/DTO/ResponseDto/RoomTypeResponseDto.cs]: RoomType lar qanaqa boladigan boldi? RoomType enum bolishi kerakmasmidi? DTOda description ham bolishi kerak turi bilan birga. masalan single room, double room. Buni modeli o'zgartirish kerak birinchi

**Review by SheraliErkayev — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/DTO/RequestDto/EmployeeRequestDto.cs (hunk 1 above)
Comment (isfandiyoro5) [reply TestHotel/TestHotel.DataAccess/DTO/RequestDto/EmployeeRequestDto.cs]: qilindi

File: TestHotel/TestHotel.DataAccess/DTO/ResponseDto/EmployeeResponseDto.cs (hunk 2 above)
Comment (isfandiyoro5) [reply TestHotel/TestHotel.DataAccess/DTO/ResponseDto/EmployeeResponseDto.cs]: bajarildi

File: TestHotel/TestHotel.DataAccess/DTO/ResponseDto/GuestResponseDto.cs (hunk 3 above)
Comment (isfandiyoro5) [reply TestHotel/TestHotel.DataAccess/DTO/ResponseDto/GuestResponseDto.cs]: korib chiqiladi

File: TestHotel/TestHotel.DataAccess/DTO/ResponseDto/RoomTypeResponseDto.cs (hunk 4 above)
Comment (isfandiyoro5) [reply TestHotel/TestHotel.DataAccess/DTO/ResponseDto/RoomTypeResponseDto.cs]: bu ham qilindi

**Review by MirazamTUT — APPROVED** [review]

**Review by Bekhzod7799 — APPROVED** [review]

**Review by SheraliErkayev — APPROVED** [review]

**Pull request merged**